Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -404,6 +404,60 @@ for armv7.
 Attaching to process 2670...
 ```
 
+###### Launching an application via debugserver
```

Comment:
Instead of going so deep with the titles (this is now going 6 levels deep), just make a sentence instead:

With the following command you can launch an application via debugserver: